api/src/main/resources/messages_fr.properties

```diff
@@ -49,4 +49,22 @@ ${project.parent.artifactId}.patienthistory.demographics.gender=Sexe
 ${project.parent.artifactId}.patienthistory.demographics.fulladdress=Adresse
 ${project.parent.artifactId}.patienthistory.requestedby=Initié par
 ${project.parent.artifactId}.patienthistory.page=Page
-${project.parent.artifactId}.patienthistory.of=de
\ No newline at end of file
+${project.parent.artifactId}.patienthistory.of=de
```

Comment:
This got me thinking, if `newEpisodesOfDiseases` is camel case'd (which is fine), then others should also be.

Reply:
I'd do this in separate PR since it'd result in some amount of rework.